Packages/com.unity.inputsystem/InputSystem/Plugins/UI/InputSystemUIInputModule.cs

```diff
@@ -162,6 +162,46 @@ public override bool IsPointerOverGameObject(int pointerOrTouchId)
             return m_PointerStates[stateIndex].eventData.pointerEnter != null;
         }
 
+        /// <summary>
+        /// Returns the most recent raycast information for a given pointer or touch.
+        /// </summary>
+        /// <param name="pointerOrTouchId">ID of the pointer or touch. Meaning this should correspond to either
+        /// <c>PointerEventData.pointerId</c> or <see cref="ExtendedPointerEventData.touchId"/>. The pointer ID
+        /// generally corresponds to the <see cref="InputDevice.deviceId"/> of the pointer device. An exception
+        /// to this are touches as a <see cref="Touchscreen"/> may have multiple pointers (one for each active
+        /// finger). For touch, you can use the <see cref="TouchControl.touchId"/> of the touch.
+        ///
+        /// Negative values will return an invalid <see cref="RaycastResult"/>.</param>
+        /// <returns>The most recent raycast information.</returns>
+        /// <remarks>
+        /// This method is for the most recent raycast, but depending on when it's called is not guaranteed to be for the current frame.
+        /// This method can be used to determine raycast distances and hit information for visualization.
+        ///
+        /// Use <see cref="RaycastResult.isValid"/> to determine if pointer hit anything.
```

Comment:
Does the `<see` to `RaycastResult` here and above actually work? References to stuff outside the package are painful ATM.

Reply:
My Rider doesn't seem to support <See> links anywhere, but just checked in in VS 2019, and the RaycastResult and isValid both sent me to the right location.

Reply:
Unfortunately, the IDE side will paint an incorrect picture there. On the Unity side, it's the output from the  `doctools` package which will have to get the link right. We have a new test on `develop` that checks for XMLDoc errors in the generated, which hopefully, if there's anything wrong it catches that.